### ISSUE TITLE
Accessibility issue: All page content should be contained by landmarks on /vet/Leptospirosis2021.htm

### ISSUE THREAD

**Issue by Alexz1008** [Alexz1008/LACODPHTest#7]
## What
  An accessibility scan flagged the element `<a href="#" id="google_translate_element-show" class="translateBtn" onclick="showTranslate(&apos;google_translate_element&apos;);return false;"><i class="fa fa-globe fa-lg" aria-hidden="true"></i>  Translate</a>` on http://www.ph.lacounty.gov/vet/Leptospirosis2021.htm because all page content should be contained by landmarks. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/region?application=playwright.

  
  To fix this, ensure all page content is contained by landmarks.
  
Specifically:

Fix any of the following:
-   Some page content is not contained by landmarks

  ## Acceptance Criteria
  - [ ] The specific axe violation reported in this issue is no longer reproducible.
  - [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
  - [ ] A test SHOULD be added to ensure this specific axe violation does not regress.
  - [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
